OrbitQt/orbittablemodel.cpp

```diff
@@ -65,10 +75,10 @@ QVariant OrbitTableModel::headerData(int section, Qt::Orientation orientation,
 QVariant OrbitTableModel::data(const QModelIndex& index, int role) const {
   if (role == Qt::DisplayRole) {
     return QVariant(QString::fromStdWString(
-        m_DataView->GetValue(index.row(), index.column()).c_str()));
+        m_DataView->GetValue(index.row(), index.column())));
   } else if (role == Qt::BackgroundRole) {
     if (m_AlternateRowColor) {
-      if (index.row() & 1)
+      if (index.row() % 2 == 1)
```

Comment:
I'm curious, is that for readability or did this produce a warning?

Reply:
This is likely not a concern here, but FYI:
 imod performs terribly on many ARM platforms. (Although since this is a const, probably gets turned into & anyway)

Reply:
This was really only for readability (not that it wasn't readable, it's more of a preference and I ended up doing it automatically). To be fair, clang-tidy (CLion shows me clang-tidy warnings by default) complains when using binary bitwise operators with a signed operand (index.row), but I don't try to honor this one specifically as it shows up too often.